bip-musig2.mediawiki

```diff
@@ -0,0 +1,824 @@
+<pre>
+  BIP: ?
+  Title: MuSig2
```

Comment:
Something more here might be nice

Reply:
Just clarifying: You're saying that the title could be more elaborate?

Reply:
Yes, with just this, nobody knows what it is

Reply:
updated